### PR TITLE
Private public info

### PR DESCRIPTION
**Pull request by Neloop**

### CODE REVIEW

**Comment by Neloop** [comment]
Can try...

File: app/V1Module/presenters/InstancesPresenter.php [hunk 1]
```diff
@@ -178,30 +184,11 @@ public function actionGroups(string $id) {
...
-
-  /**
-   * Get a list of all public groups in an instance.
-   * @GET
-   * @param string $id An identifier of the instance
-   * @throws ForbiddenRequestException
-   */
-  public function actionPublicGroups(string $id) {
-    $instance = $this->instances->findOrThrow($id);
-    if (!$this->instanceAcl->canViewGroups($instance)) {
```
Comment (Teyras) [inline app/V1Module/presenters/InstancesPresenter.php]: Doesn't removing this mean that we can remove the `canViewGroups` privilege too?

Comment (Neloop) [reply app/V1Module/presenters/InstancesPresenter.php]: Not really sure... maybe... I'll look into it

Comment (Teyras) [reply app/V1Module/presenters/InstancesPresenter.php]: I see. Maybe stuff like [this](https://github.com/ReCodEx/api/blob/master/app/V1Module/presenters/InstancesPresenter.php#L179) should use GroupViewFactory too?

Comment (Neloop) [reply app/V1Module/presenters/InstancesPresenter.php]: You mean... like this: https://github.com/ReCodEx/api/blob/private-public-info/app/V1Module/presenters/InstancesPresenter.php#L191 ... ?

Comment (Teyras) [reply app/V1Module/presenters/InstancesPresenter.php]: Yeah, sorry for not checking the right branch.

**Pull request closed without merging**